howto/change-python.md

```diff
@@ -0,0 +1,32 @@
+# Bump Python Version for DjangoGirls.org Website
+
+In order to bump up the Python version for the DjangoGirls.org website, you 
+need to have access to our PythonAnywhere account, and follow these instructions:
+
+1. [Login to PythonAnywhere](https://www.pythonanywhere.com/user/djangogirls2/consoles/). Credentials can be found in 1Password.
+2. Check if the Python version you want to upgrade to is supported by the current
+system image used on PythonAnywhere by clicking on the pen icon on the Web Tab or
+by [reading this](https://help.pythonanywhere.com/pages/ChangingSystemImage#:~:text=Once%20you%20have%20changed%20the,again%20to%20pick%20it%20up.).
+ If yes, no need to change the system image. If system image needs to be changed, follow the instructions here.
+3. Check which node version we are using opening the `Bash Console` and type 
+```which node```
+4. Change the system image if necessary (skip this step if version is supported by
+current image).
+5. Navigate to the ```.virtualenvs``` folder.
+6. Create a new virtual environment.
+7. Navigate to the ```djangogirls.com``` folder and run ```pip install -r requirements.txt```. 
+This ensures that even if when you get to run the ```deploy.sh``` script and it fails,
+you still have a working virtual environment with all the packages required to run the website.
```

Comment:
To be able to follow this conclusion / argumentation, one probably needs to know more about the `deploy.sh` script. Does that script also install these packages and this is giving the reason why (thus redundantly) installing them manually here, too, is a good idea? What advantage is there in having a usable virtual environment when the script fails? Can it be used for debugging?

Reply:
We are using gulp to manage static files which is something we are currently looking into. The node bit of the deploy script may fail and since static files are already compiled, the website will still work while that bit is failing. Running the deploy script after setting up everything is just a debugging step to see if all is set up correctly. If there are issues to be resolved, for example, like gulp issues, they can be attended to without further downtime.